tests/nlu/test_evaluation.py

```diff
@@ -448,9 +440,12 @@ def test_run_cv_evaluation(
         assert all(key in extractor_evaluation for key in ["errors", "report"])
 
 
-def test_run_cv_evaluation_with_response_selector(monkeypatch: MonkeyPatch):
+@pytest.mark.timeout(
```

Comment:
how about a test where there are no entities in the test set / none are predicted?